pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-events/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, pidgin, glib, autoreconfHook, pkgconfig, intltool }:
+
+stdenv.mkDerivation rec {
+  pname = "purple-events";
+  version = "2017-08-25";
+
+  src = fetchFromGitHub {
+    owner = "sardemff7";
+    repo = pname;
+    rev = "7b01b0a2724f675dadf41a8d8e9bee203ef1ccc2";
+    sha256 = "1mhg40swa12inkdhhxf6x9c45h2dijzvdlx1325zgvs2k1vv0a3k";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig intltool ];
+  autoreconfPhase = "${stdenv.shell} ./autogen.sh";
+  buildInputs = [ pidgin glib ];
+
+  configureFlags = [ "--with-purple-plugindir=${placeholder "out"}/lib/purple-2" ];
+
+  meta = with stdenv.lib; {
+    homepage = http://purple-events.sardemff7.net/;
```

Although upstream uses this URL as it's website as in the GitHub repo description, it gets redirected to the GitHub page, implying perhaps we should use the following:

```suggestion
    homepage = "https://github.com/sardemff7/purple-events";
```

Note the quotes added.